react-relay/index.d.ts

```diff
@@ -128,7 +146,13 @@ declare module "react-relay" {
     }
 
     interface RelayProp {
-        variables: any
-        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void
+        route: { name: string; }; // incomplete, also has params and queries
+        variables: any;
+        pendingVariables?: any;
+        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        forceFetch(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        hasOptimisticUpdate(record?: any): boolean;
+        getPendingTransactions(record?: any): RelayMutationTransaction[];
+        commitUpdate?: (mutation: Mutation<any,any>, callbacks?: StoreUpdateCallbacks<any>) => any;
```

Comment:
That's actually entirely fair. It will always be available.

I've been working with relay subscriptions a fair bit and they require a wrapper to be present; not so with commitUpdate for mutations.